rslib/src/import_export/package/apkg/import.rs

```diff
@@ -426,8 +432,7 @@ impl<'a> Context<'a> {
 
     fn import_revlog(&mut self) -> Result<()> {
         for mut entry in mem::take(&mut self.data.revlog) {
-            if let Some(cid) = self.remapped_cards.get(&entry.cid) {
-                entry.cid = *cid;
+            if self.added_cards.contains(&entry.cid) {
```

Comment:
Just to double-check we're on the same page here, the way the Python code works is:

- a given card is imported if its (note.guid, ordinal) isn't in the target collection
- it's assigned a new ID if its existing one conflicts with an ID in the collection
- because it may have a new ID, the revlog entries that reference it need to be updated to point to the new card id

It looks like the cid update has been removed here; are you planning to handle it a different way?

Reply:
See above. Thanks for reiterating. I came to the conclusion that we don't need the guid here. Either the card's note could not be imported, then its original id is in `conflicting_notes`. Or it was imported successfully, then its possibly remapped id is unique in the target collection.